api/src/shared/infrastructure/repositories/assessment-repository.js

```diff
@@ -101,15 +101,11 @@ async function _getAssociatedCampaign(campaignParticipationId) {
 }
 
 const abortByAssessmentId = function (assessmentId) {
-  return this._updateStateById({ id: assessmentId, state: Assessment.states.ABORTED });
+  return _updateStateById({ id: assessmentId, state: Assessment.states.ABORTED });
```

Comment:
Je ne comprends pas comment cela pouvait fonctionner auparavant 🤔 
A quoi faisait référence le `this` ?